pkgs/applications/networking/instant-messengers/twitch-tui/default.nix

```
@@ -0,0 +1,25 @@
+{ openssl, lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "twitch-tui";
+  version = "v1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "Xithrius";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-144yn/QQPIZJOgqKFUWjB7KCmEKfNpj6XjMGhTpQdEQ=";
+  };
+
+  OPENSSL_LIB_DIR = "${lib.getLib openssl}/lib";
+  OPENSSL_INCLUDE_DIR = "${openssl.dev}/include";
```

```suggestion
  nativeBuildInputs = [ pkg-config ];
  buildInputs = [ openssl ];
```

That's what pkg-config is for. :smiley_cat: